### DISCUSSION TITLE
3rd party risk management

### DISCUSSION THREAD

**Discussion by risingfish** [jgraph/drawio#4181] [General]
I've been trying to get app.diagrams.net approved for use in my employer. We're a fintech company so there's pretty strict controls on what software we can use. Even though app.diagrams.net is free, I've been told that it must still be run through our Third Party Risk Management process. Are the app.diagrams.net devs familiar with TPRM and willing to go through that for a potential user of the product?

**Comment by alderg** [comment]
The way most companies handle this is to deploy the app internally within their network. diagrams.net is effectively 2 developers, we don't have the capacity to do security reviews.

**Reply by risingfish** [reply]
Ok, that's what I was expecting, but I had to ask. Thank you for the quick response! I suppose I will see if there is an appetite within our org to stand it up internally.
